### DISCUSSION TITLE
Add new option to "Perfect" mod to miss instead of failing the map.

### DISCUSSION THREAD

**Discussion by Trung0246** [ppy/osu#20542] [Ideas]
Currently the only option is to restart on fail like "Sudden Death". However unlike SD, Perfect could have option to count non-300 hit as a miss instead.

Since restart on fail and count as miss are mutually exclusive therefore I suggest to change this to dropdown list instead of option radio button. However this is probably unnecessary.

With the introduction of "count as miss" I would think that "restart on fail" for Perfect mod will become unnecessary as Perfect mod with count as miss behavior can be combined with Sudden Death to emulate the old Perfect mod behavior. If we go with this way then we probably should change the description from "SS or quit" to "SS or miss" or something similar.

**Comment by nagi-desuuu** [comment]
So what you're proposing is to shorten the hit window significantly so that other types of judgements (ok  meh, etc.) are null and void, am I right?

**Reply by Trung0246** [reply]
Yeah that the gist to it. Optionally I think combining the proposed behavior with SD can emulate the old Perfect behavior.

**Comment by peppy** [comment]
If this is to be implemented as an option on perfect mod, it should first be ensured that perfect mods converts non-perfect hits to misses. Then the setting which is added should be "fail on non-perfect hit" or something like that.

**Reply by ataman6** [reply]
So will it be implemented?
P.S. I had the exact same idea as the author and I found this page.
